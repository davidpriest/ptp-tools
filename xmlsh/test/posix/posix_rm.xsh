# Test rm
import package posix=org.xmlsh.commands.posix

posix:rm -rf $TMPDIR/_xmlsh
posix:mkdir $TMPDIR/_xmlsh

# Create initial file
posix:cat < ../../samples/data/books.xml > $TMPDIR/_xmlsh/f1

cd $TMPDIR/_xmlsh

posix:cp f1 f2

[ -f f1 -a -f f2 ] || echo Fail invalid start condition
posix:rm f1
[ -f f1 ] || echo Success f1 deleted
posix:rm f2 
[ -f f2 ]|| echo Success f2 deleted

cd ..
posix:rm -rf $TMPDIR/_xmlsh
