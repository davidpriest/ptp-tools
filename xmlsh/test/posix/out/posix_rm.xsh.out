Success f1 deleted
Success f2 deleted
